cmds/grpc-backend/main.go

```diff
@@ -173,6 +175,14 @@ func main() {
 		ctx    = context.Background()
 		logger = logging.WithValuesFromContext(ctx, logging.Logger)
 	)
+	if *profServiceName != "" {
+		if err := profiler.Start(
+			profiler.Config{
+				Service:        *profServiceName,
+				ServiceVersion: "v1"}); err != nil {
+			logger.Error("Failed to start the profiler ", zap.Error(err))
```

Comment:
Seems like we should rather be strict here and use `logger.Panic`, instead. With that, any issues with setting up the profiling become very visible.

Reply:
Done..